ContosoHelpdeskChatBot/ContosoHelpdeskChatBot/Dialogs/ResetPasswordDialog.cs

```diff
@@ -54,10 +54,10 @@ private async Task<DialogTurnResult> PasscodeReceivedStepAsync(WaterfallStepCont
                 {
                     string temppwd = "TempPwd" + new Random().Next(0, 5000);
                     using (var db = new ContosoHelpdeskContext())
-                    {​
```

Comment:
this is a known issue and it is getting fixed in master as well as other branches.